src/cdp-proxy/CDPMessageHandlers/directCDPMessageHandler.ts

```diff
@@ -1,13 +1,99 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT license. See LICENSE file in the project root for details.
 
-import { ICDPMessageHandler, ProtocolMessage } from "./ICDPMessageHandler";
+import { Protocol as Cdp } from "devtools-protocol/types/protocol";
+import { ICDPMessageHandler, ProcessedCDPMessage } from "./ICDPMessageHandler";
 
 export class DirectCDPMessageHandler implements ICDPMessageHandler {
 
+    /**
+     * @description The Hermes native functions calls mark in call stack
+     * @type {string}
+     */
+    private readonly HERMES_NATIVE_FUNCTION_NAME: string = "(native)";
+
+    /**
+     * @description Equals to 0xfffffff - the scriptId returned by Hermes debugger, that means "invalid script ID"
+     * @type {string}
+     */
+    private readonly HERMES_NATIVE_FUNCTION_SCRIPT_ID: string = "4294967295";
+
+    private readonly ARRAY_REQUEST_PHRASE_MARKER: string = "Object.getOwnPropertyDescriptor";
+
     constructor() { }
 
-    public processCDPMessage(evt: any): ProtocolMessage {
-        return evt;
+    public processDebuggerCDPMessage(event: any): ProcessedCDPMessage {
+        let reverseDirection = false;
+        if (event.method === "Debugger.setBreakpoint") {
```

Comment:
I suppose that we can enumerate all used debugger constant string literals as the code objects, like we process the errors https://github.com/microsoft/vscode-react-native/blob/master/src/common/error/errorStrings.ts